CONTRIBUTING.md

```diff
@@ -61,9 +61,48 @@ If you want to verify the changelog entries as described by a range of Git revis
 
     ./unreleased.sh master..
 
+## Writing a good changelog entry
+
+Writing good changelog entries is **important**: as a developer, it gives visibility on your contribution; as a user, it makes clear what is new, what changes and two deal with those, making the product more accessible and your work more meaningful.
+
+The raw changelog is used to compile a meaningful summary of changes across releases. This happens some time after the PR has been merged and the person taking the responsability of summarizing new user-facing features must be in the position of easily understand the nature of the change and report it. The ideal changelog entry can be more or less incorporated verbatim in the release notes.
+
+Here are a few practical tips
+
+* the first term to appear should be the affected component -- [here's a list](#list-of-components-for-changelog-entries)
+* write as many as necessary changelog entries as necessary
+* don't be _too_ succinct: a single entry does **not have to** fit on a single line
+* on the other end, if the size grows beyond 5-6 lines, rather add a link to a relevant documentation or issue with more details
+* the ultimate target are end users: focus on the impact on them, tell them what's new or how to deal with a change
```

Comment:
```suggestion
* the ultimate target are end users: focus on the impact on them, tell them what's new or how to deal with a change. This also means that purely internal changes should not have a release note entry since they are not relevant to users.
```

Reply:
@cocreature: These are not sentences; adding periods makes it unbalanced.

```suggestion
* the ultimate target are end users: focus on the impact on them, tell them what's new or how to deal with a change
* avoid writing entries for purely internal changes, since they are not relevant to users
```

Reply:
I'd rather keep points short, I'll make this a separate one. It's a re-iteration of what stated above, but re-iteration makes sense here. :slightly_smiling_face:

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6666/commits/2184613e2d05957ee352ed806c94a0a5899b5de8